{{cookiecutter.project_slug}}/.github/workflows/build.yml

```diff
@@ -0,0 +1,41 @@
+name: Build ${{ cookiecutter.project_slug }}
+
+on: [push, pull_request]
+
+jobs:
+
+  build:
+    name: Build for (${{ matrix.python-version }}, ${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
+        python-version: ['3.6', '3.7', '3.8']
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Python info
+        shell: bash -l {0}
+        run: |
+          which python
+          python --version
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Check style against standards using prospector
+        shell: bash -l {0}
+        run: prospector -o grouped -o pylint:pylint-report.txt
+      - name: Build
+        shell: bash -l {0}
+        run: |
+          python setup.py build
+      - name: Test
+        shell: bash -l {0}
+        run: |
+          pip install pytest
```

Comment:
Instead of installing pytest like this we could miss some deps maybe we can install the deps listed at 
 setup.py:test_requires

Reply:
maybe better to just use `python setup.py test`

Reply:
done

Reply:
This will run tests twice, but thats better than hardcoding the test requirements in the ci config file.